tests/e2e/tests/regression/browser/database-info.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import {
+    MyRedisDatabasePage,
+    WorkbenchPage,
+    BrowserPage
+} from '../../../pageObjects';
+import { rte } from '../../../helpers/constants';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const workbenchPage = new WorkbenchPage();
+const browserPage = new BrowserPage();
+
+fixture `Database info tooltips`
+    .meta({type: 'regression'})
+    .page(commonUrl)
+    .beforeEach(async () => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async () => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+test
+    .meta({ rte: rte.standalone })
+    ('Verify that user can see an (i) icon next to the database name on Browser and Workbench pages', async t => {
+        await t.expect(browserPage.databaseInfoIcon.visible).ok('User can see (i) icon on Browser page');
+        //Move to the Workbench page and check icon
+        await t.click(myRedisDatabasePage.workbenchButton);
+        await t.expect(workbenchPage.overviewTotalMemory.visible).ok('User can see (i) icon on Workbench page');
```

Comment:
I suggest setting a timeout within the assertion.

Reply:
added